README.adoc

```diff
@@ -157,18 +157,18 @@ have IPv4 addresses, the address on eth2 is used.
 === HTTP Proxy Settings
 
 In an environment where the HTTP traffic needs to pass through an
-HTTP proxy server, set the `http_proxy`, `http_proxy_user` and
-`http_proxy_password` proxy configurations in the Vagrantfiles to enable
+HTTP proxy server, set the `proxy`, `proxy_user` and
```

Comment:
@hferentschik It seems vagrant-registration does not use `http` prefix , so may be we should get rid of it and be consistent.